docs/src/convergence.md

```diff
@@ -0,0 +1,181 @@
+# [Experimental convergence order of MPRK schemes](@id convergence_mprk)
+
+In this tutorial we check that the implemented MPRK schemes have the expected order of convergence. 
+
+## Conservative production-destruction systems
+
+First, we consider conservative production-destruction systems (PDS). To investigate the convergence order we define the non-autonomous test problem 
+
+```math
+\begin{aligned}
+u_1' &= \cos(\pi t)^2 u_2 - \sin(2\pi t)^2 u_1, & u_1(0)&=0.9\\
+u_2' & = \sin(2\pi t)^2 u_1 - \cos(\pi t)^2 u_2, & u_2(0)&=0.1
+\end{aligned}
+```
+for ``0≤ t≤ 1``.
+The PDS is conservative, since the sum of the right hand side terms equals zero. 
+An implementation of the problem is given next.
+
+
+```@example eoc
+using PositiveIntegrators
+
+# define problem
+P(u, p, t) = [0.0 cos.(π * t) .^ 2 * u[2]; sin.(2 * π * t) .^ 2 * u[1] 0.0]
+prob = ConservativePDSProblem(P, [0.9; 0.1], (0.0, 1.0))
+
+nothing # hide
+```
+
+To use `analyticless_test_convergence` from [`DiffEqDevTools`](https://github.com/SciML/DiffEqDevTools.jl) we need to pick a solver to compute the reference solution and specify tolerances. Since the problem is not stiff we use the high order explicit solver `Vern9()` from [`OrdinaryDiffEq`](https://docs.sciml.ai/OrdinaryDiffEq/stable/). Moreover, we need to choose the different time step sizes which are used to investigate the convergence behavior. 
```

Comment:
```suggestion
To use `analyticless_test_convergence` from [DiffEqDevTools.jl](https://github.com/SciML/DiffEqDevTools.jl), we need to pick a solver to compute the reference solution and specify tolerances.
Since the problem is not stiff, we use the high-order explicit solver `Vern9()` from [OrdinaryDiffEqVerner.jl](https://docs.sciml.ai/OrdinaryDiffEq/stable/).
Moreover, we choose time step sizes to investigate the convergence behavior. 
```